lib/google/ads/google_ads/logging_interceptor.rb

```diff
@@ -29,9 +29,15 @@ def initialize(logger)
         end
 
         def request_response(request:, call:, method:, metadata: {})
+          customer_id = "N/A"
```

Comment:
I was reviewing the Python version of this (https://github.com/googleads/google-ads-python/pull/262#discussion_r407575545). Are you sure that "N/A" is the correct default response given that '/' is the delimiter between resource name parts? Perhaps "NA" or "None"?